scripts/install_xcpng.py

```diff
@@ -43,7 +43,7 @@ def generate_boot_conf(directory, installer, action):
 {rt}
 """)
 
-def generate_answerfile(directory, installer, hostname_or_ip, target_hostname, action, hdd):
+def generate_answerfile(directory, installer, hostname_or_ip, target_hostname, action, hdd, ngc):
```

Comment:
I would prefer if we named the variable `netinstall_gpg_check` everywhere. Sure, its a bit long, and you may have to split some lines to avoid pycodestyle complaints, but it's more readable.